### PR TITLE
U iv2

### PR DESCRIPTION
**Pull request by duquettelr**
pulling UI changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+
+            <div class="row right" class="right hide-on-med-and-down">
+                <a href="SigninAuth.html" id="download-button" class="btn-large waves-effect waves-light orange">User
+                    Login</a>
+            </div>
+            <ul id="nav-mobile" class="right hide-on-med-and-down">
+                <li><a href="sass.html">Exercises</a></li>
+                <li><a href="badges.html">FAQ</a></li>
+                <li><a href="collapsible.html">The Team</a></li>
```
Comment (alexjzyang) [inline index.html]: sass.html, badges.html, collapsible.html don't exist, could we please put a "#" without quotes here?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+    <footer class="page-footer orange">
+        <div class="container">
+            <div class="row">
+                <div class="col l6 s12">
+                    <!-- donation button goes here -->
+                    <a href="https://act.autismspeaks.org/site/Donation2?idb=135287805&4141.donation=form1&DONATION_LEVEL_ID_SELECTED=1&df_id=4141&mfc_pref=T&idb=[[S76:idb]]"
+                        button class="btn waves-effect waves-light" type="submit" name="action">Donate to Autism Speaks
+                        <i class="material-icons right">monetization_on</i>
+                        </button></a>
```
Comment (alexjzyang) [inline index.html:144]: Is the donation button color intentionally greyed out? Or do we need a little more contrast?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        <!-- site title -->
+        <div class="nav-wrapper container"><a id="logo-container" href="#" class="brand-logo">NowYou!</a>
+            <!-- link to login page -->
+
+
+            <div class="row right" class="right hide-on-med-and-down">
+                <a href="SigninAuth.html" id="download-button" class="btn-large waves-effect waves-light orange">User
+                    Login</a>
+            </div>
+            <ul id="nav-mobile" class="right hide-on-med-and-down">
```
Comment (alexjzyang) [inline index.html:31]: Hide nav on mobile size here is great, but the side nav doesn't appear on small screen sizes. I think we're missing side nav button and associated side-nav class.
Let me know if you need any help there.

File: js/init.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+(function($){
+  $(function(){
+
+    $('.sidenav').sidenav();
+
+  }); // end of document ready
+})(jQuery); // end of jQuery name space
```
Comment (alexjzyang) [inline js/init.js:7]: 1. We do not have .sidenav class in index.html
2. Do we know the exact functionality of (function($){...})(jQuery) here?
I normally build from what I know, trimming down copied code to bare minimal. But if there's a specific reason why it's here, then please ignore my comment.

**Review by alexjzyang — CHANGES_REQUESTED** [review]
Hi Paulina,
Great job! I love the UI you designed. Just a few minor comments here and there, if you could briefly address them.
Also Could you put css and js folder into assets folder? Just to clean things up a little bit.
Thanks
Alex

File: index.html (hunk 2 above)
Comment (PaulinaBootcamp) [reply index.html:144]: we shall style the button at a later time :)

**Review by alexjzyang — APPROVED** [review]

**Pull request merged**